### DISCUSSION TITLE
[BP#85] Mettre en place un plan de fin de vie

### DISCUSSION THREAD

**Discussion by ddebarbieux** [cnumr/best-practices#238] [Bonnes pratiques] [answered]
Voici un extrait de la description de cette BP :
_Le plan de fin de vie consiste à s’assurer que le jour où un site, une application métier interne, ou un service en ligne n’a plus lieu d’être ou n’est plus utilisé, il sera aussitôt décommissionné._

Résumer la fin de vie à décommissionner des services me semble léger. Je propose donc de 

1.	Soit changer le titre pour y faire apparaitre le mot ‘décommissionné’.[ Une discussion sur le sujet a déjà lieu dans la BP#89](https://github.com/cnumr/best-practices/discussions/154).
2.	Soit garder le titre, mais être plus ambitieux dans notre vision d’un plan de fin de vie.

**Comment by tbroyer** [comment]
> Résumer la fin de vie à décommissionner des services me semble léger.

Qu'est-ce que tu ajouterais ?

**Reply by ddebarbieux** [reply]
1. Identifer la fin de vie en mesurant le taux d'utilisation d'un service / d'une fonctionalitée (cf BP#1)
2. Parler de migration : c'est le cas de fin de vie qui me semble le plus fréquent
3. Une application 'client lourd Windows' est livré avec un desinstalleur. Quel est l'équivalent pour l'éco-conception Web ?

J'ai plein d'idées et de concepts. Malheuresement, rien de très concret...

**Reply by ACTLEM** [reply]
Le deuxième point me parle beaucoup. Les fameux projets de pour passer sur une nouvelle stack "hype" mais "iso-fonctionnel" sans se poser la question de l'apport technique (une "simple" migration technique suffirait ou pas?) ou sur les fonctionnalités non utilisées et que l'on reprend au cas où...et à celà s'ajoute la période souvent longue de transition. J'ai malheureusement trop d'exemples vécus...

**Reply by ACTLEM** [reply]
On partirait sur un exemple ou ajout dans la description du point sur la migration qui est un cas fréquent sur le terrain. Votre avis?

**Comment by ACTLEM** [comment, marked as answer]
Issue pour amélioration de la BP pour parler des conditions d'arrêt, de migration, et comment on met fin à un service: https://github.com/cnumr/best-practices/issues/316
